### DISCUSSION TITLE
Updating SaltStack?

### DISCUSSION THREAD

**Discussion by scarl3t0** [Security-Onion-Solutions/securityonion#6304] [Unsupported Versions] [answered]
Hi all,

I'm running a distributed deployment of SO 2.3.80. I installed the manager and search node on clean VMs, and the sensor on a clean physical server (all using the ISO). My vulnerability scanner reports a high severity vuln in SaltStack < 3003.3 with the installed version being 3003.1.e17. Would it be possible to update SaltStack separately without breaking the ever-loving heck out of my install?

Thanks in advance,

scarl3t0

**Comment by dougburks** [comment, marked as answer]
> Would it be possible to update SaltStack separately without breaking the ever-loving heck out of my install?

No, we don't support manually updating SaltStack. We have plans to update SaltStack soon. In the meantime, the compensating controls are that the local host-based firewall limits access to the salt ports and they are only exposed to known good Security Onion nodes.
